setup.py

```diff
@@ -15,7 +21,8 @@
     url='https://github.com/civisanalytics/muffnn',
     description=('Multilayer Feed-Forward Neural Network (MuFFNN) models with '
                  'TensorFlow and scikit-learn'),
-    long_description=open(os.path.join(THIS_DIR, 'README.md')).read(),
+    long_description=read('README.md'),
     include_package_data=True,
-    license="BSD-3"
+    license="BSD-3",
+    install_requires=read('requirements.txt').split('\n')
```

Comment:
this [doesn't work in general](http://stackoverflow.com/a/16624700) but seems fine for our requirements.txt